doc/nrf/includes/wifi_credentials_shell.txt

```diff
@@ -0,0 +1,21 @@
+Configuring Wi-Fi access point credentials
+==========================================
+
+This sample uses the :ref:`lib_wifi_credentials` library to manage Wi-Fi credentials.
+Before the sample can connect to a Wi-Fi network, you must configure at least one credential set.
+
+Once your device has been flashed with this sample, connect to your device's UART interface and add credentials using the following command:
+
+.. parsed-literal::
+   :class: highlight
+
+   wifi_cred add *NetworkSSID* WPA2-PSK *NetworkPassword*
+
+Where *NetworkSSID* is replaced with the SSID of the Wi-Fi access point you want your device to connect to, and *NetworkPassword* is its password.
+Then either reboot the device or use the ``wifi_cred auto_connect`` command to manually trigger a connection attempt.
+
+From now on, these credentials will automatically be used when the configured network is reachable.
+
+Enabling the :kconfig:option:`CONFIG_WIFI_CREDENTIALS_BACKEND_PSA` Kconfig option instead of using :kconfig:option:`CONFIG_WIFI_CREDENTIALS_BACKEND_SETTINGS` stores Wi-Fi credentials in a PSA protected storage, adding more security.
```

Comment:
I suggest that we set the default value of the wifi credentials PSA backend to y when building for non-secure wifi builds.
And update the comment to say something like.

When building for non-secure the wifi credentials backend will be set to PSA using TFM. This ensures that credentials are stored to and retreived from protected storage, adding more security.

Reply:
ie.

```
config WIFI_CREDENTIALS_BACKEND_PSA
	bool "PSA Protected Storage"
	depends on BUILD_WITH_TFM && !TFM_PROFILE_TYPE_MINIMAL
	default y
```